.github/workflows/test-docker-deployments.yml

```diff
@@ -0,0 +1,36 @@
+name: Test Deployments
+on:
+  workflow_dispatch:
```

Comment:
i'm not convinced it's actually worthwhile to add this to every PR. I'm more like 60/40 that it feels like this is better as a test that's run prior to publishing artifacts and not every PR. I don't merge to ziti enough to know how annoying this test will be ... probably best to ask the group to see what they think.

Reply:
I aim to surface issues as early as practical in the life cycle. Advertised this change to the team in case it's perceived as a problem.